onnxruntime/core/session/abi_session_options.cc

```diff
@@ -130,7 +130,7 @@ ORT_API_STATUS_IMPL(OrtApis::SetSessionGraphOptimizationLevel, _In_ OrtSessionOp
       options->value.graph_optimization_level = onnxruntime::TransformerLevel::Level2;
       break;
     case ORT_ENABLE_ALL:
-      options->value.graph_optimization_level = onnxruntime::TransformerLevel::Level3;
```

Comment:
How do users enable level 3 after this change if they don't want level 4?

Reply:
It is not able to do that unless we change public API, current design has only three buckets. Internally, we could have many levels that mapped to these 3 buckets.